pkgs/by-name/pa/pandoc/package.nix

```diff
@@ -1,13 +1,14 @@
-{ stdenv, lib, haskellPackages, haskell, removeReferencesTo, installShellFiles }:
+{ stdenv, lib, haskellPackages, haskell, removeReferencesTo, installShellFiles, pandoc-cli ? (p: p.pandoc-cli) }:
```

Comment:
Nit: I don't like the fact that the `pandoc-cli` arg looks like a package input when it's actually a selector function. I'd make this more obvious by calling it e.g. `selectPandocCLI`.

A brief doc comment would also be nice.